src/main/java/com/leverx/learningmanagementsystem/btp/registry/constants/RegistryConstants.java

```diff
@@ -0,0 +1,22 @@
+package com.leverx.learningmanagementsystem.btp.registry.constants;
+
+import lombok.NoArgsConstructor;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@NoArgsConstructor(access = PRIVATE)
+public class RegistryConstants {
+
+    public static final String SCHEMA = "schema_%s";
+    public static final String DB_NAME = "hana-database";
+    public static final String SERVICE_PLAN_ID = "9196d940-4ba6-452c-941a-094b13934083";
+    public static final String TENANT_ID = "tenantId";
+    public static final String BINDING = "binding_%s";
+    public static final String ROUTER_URL = "https://%s-learning-management-system-approuter.cfapps.us10-001.hana.ondemand.com";
```

Comment:
approuter name shouldn't be hardcoded and should be moved to environment variables and it can be placed in user-provided service (as it can be changed)
Additionally, space and domain can be changed and we should be prepared for this. It also can be done via environment variables and we can read it from VCAP_APPLICATION env (you can find in service bindings of your app) `${vcap.application.application_uris[0]}`
And then you can build tenant-specific URL:
```
String approuterUri = applicationUri.replace(applicationName, approuterName);
return "%s://%s-%s".formatted(HTTPS_PROTOCOL, tenantSubdomain, approuterUri);
```